optaweb-vehicle-routing-frontend/src/ui/header/Header.tsx

```diff
@@ -20,7 +20,7 @@ import NavigationWithRouter from './Navigation';
 
 const Header: React.FC = () => (
   <PageHeader
-    logo={<Brand src="./assets/images/optaPlannerLogo200px.png" alt="OptaPlanner Logo" />}
+    logo={<Brand src="./assets/images/optaplanner_logo_dk_200px.png" alt="OptaPlanner Logo" />}
```

Comment:
Nitpick: given the logo names here https://github.com/kiegroup/optaplanner/tree/master/optaplanner-docs/src/main/asciidoc/shared
and presuming that optaplanner_logo_dk_200px.png is the same as optaPlannerLogoDarkBackground at a different resolution, I would maybe expect optaPlannerLogoDarkBackground200px.png

Reply:
OK, aligning with optaplanner-docs.